src/explorer/SiteTreeItem.ts

```diff
@@ -170,6 +175,29 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
             workspace.getConfiguration(constants.extensionPrefix, Uri.file(fsPath)).update(constants.configurationSettings.showBuildDuringDeployPrompt, false);
         }
     }
+
+    private async promptToSaveDeployConfigs(node: IAzureNode<SiteTreeItem>, fsPath: string): Promise<void> {
+        const saveDeploymentConfig: string = `Always deploy the workspace "${fsPath}" to "${node.treeItem.client.fullName}"?`;
+        const neverShowAgain: MessageItem = { title: "Never" };
+        let result: MessageItem = DialogResponses.learnMore;
+        while (result === DialogResponses.learnMore) {
+            result = await ext.ui.showWarningMessage(saveDeploymentConfig, DialogResponses.yes, neverShowAgain, DialogResponses.skipForNow, DialogResponses.learnMore);
+            if (result === DialogResponses.yes) {
+                workspace.getConfiguration(constants.extensionPrefix, Uri.file(fsPath)).update(constants.configurationSettings.deploymentConfigurations, {
+                    app: node.id,
+                    path: fsPath
```

Comment:
You can't use the full fsPath here. Workspace settings get checked in to GitHub and one user's full fsPath won't match another user's full fsPath.

We already have a 'deploySubpath' setting. I recommend leveraging that instead of adding a new setting